src/modules/acbs/acbs-facility-fixed-fee.service.test.ts

```diff
@@ -0,0 +1,112 @@
+import { HttpService } from '@nestjs/axios';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import { AxiosError } from 'axios';
+import { when } from 'jest-when';
+import { of, throwError } from 'rxjs';
+
+import { AcbsFacilityFixedFeeService } from './acbs-facility-fixed-fee.service';
+import { AcbsGetFacilityFixedFeeResponseDto, AcbsGetFacilityFixedFeeResponseItem } from './dto/acbs-get-facility-fixed-fee-response.dto';
+import { AcbsException } from './exception/acbs.exception';
+
+describe('AcbsFacilityFixedFeeService', () => {
+  const valueGenerator = new RandomValueGenerator();
+  const idToken = valueGenerator.string();
+  const baseUrl = valueGenerator.httpsUrl();
+  const portfolioIdentifier = valueGenerator.string({ length: 2 });
+  const facilityIdentifier = valueGenerator.facilityId();
+
+  let httpService: HttpService;
+  let service: AcbsFacilityFixedFeeService;
+
+  let httpServiceGet: jest.Mock;
+
+  const expectedHttpServiceGetArgs = [
```

Comment:
Can we add assertions where `facilityIdentifier` are following inputs please:

* <String>
* ABCD
* !"£$
* ---
* ` ` (White spaces)

Reply:
We check this at a module level (if you're expecting an error to be thrown here). I've added another comment in a comment below that goes more indepth.